src/Data/Atomic.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE CPP #-}
```

Comment:
```suggestion
{-# LANGUAGE CPP                                    #-}
```

Reply:
That should align now?

Reply:
Oops, you're right. I also force pushed to fix alignment once, but I missed this one.